metaseq/modules/layer_norm.py

```diff
@@ -94,6 +115,13 @@ def get_statistics_from_all_workers(stat, rank, world_size):
         return buffer
 
     def forward(self, hidden_states):
+        if isinstance(hidden_states, ShardedTensor):
+            return torch.nn.functional.layer_norm(
+                hidden_states,
+                self.normalized_shape,
```

Comment:
I'm assuming this layer norm doesn't incur any comms?

Reply:
No this does need comms, because it's "synced."